docs/guide/write/quick-start.md

```diff
@@ -1,9 +1,69 @@
-# Coming Soon
+# Writing Tests Quick Start 
 
-::: info
-This page is Under Construction 🏗️, whilst Moonwall has been built - the docs have not.
+Let's write a super simple test case that we'll run in the 
+[Quick Start Running Tests](/guide/test/quick-start) Section. 
 
-Please Stay tuned for more details of this feature!
-:::
+## Moonwall Utils
 
-![underconstruction](/under-construction.png)
+Moonwall utils is a utils package with helpful constants and various functions for Moonwall. In our test file we want to refer to some pre-funded development accounts like ALITH and BALTATHAR which can be easily import from ```@moonwall/util```
+
+We use describeSuite to define our test suite, similar to how you would use Mocha in Javascript. 
+
+```typescript
+import {describeSuite} from "@moonwall/cli"
+import {BALTATHAR_ADDRESS, GLMR} from "@moonwall/util"
+```
+
+
+### Configure our Test Suite
+
+When describing a test suite, we need to provide an id, a title, and we need to specify the foundation that we'll be using. In this case we're using the dev foundation, so we're configuring our tests to be run against a local dev node.
+
+```typescript
+describeSuite({
+	id: "D1",
+	title: "Demo suite",
+	foundationMethods: "dev",
+	testCases: ({it, context, log})=> {
+		//Test cases will go here
+		}
+	})
+```
+
+### Write Test Cases
+
+We are going to write a basic test case that transfers some funds to Baltathar (which is prefunded account that exists on Moonbeam local dev nodes). Be sure to give the test case an id and a name that makes sense. Then we'll produce a block. 
+
+```typescript
+it ({id: "T1", title: "Demo test case", test: async()=> {
+			await context.ethers().sendTransaction({to:BALTATHAR_ADDRESS, value: 10n * GLMR });
+			await context.createBlock();
+		} })
+``` 
+
+### Sample Test File
+
+Once all is said and done your simple test file should look like the below. Remember, this is a simple demo test case and it is not refined or comprehensive. 
+
+```typescript
+import {describeSuite} from "@moonwall/cli"
+import {BALTATHAR_ADDRESS, GLMR} from "@moonwall/util"
+
+describeSuite({
+	id: "D1",
+	title: "Demo suite",
+	foundationMethods: "dev",
+	testCases: ({it, context, log})=> {
+
+		it ({id: "T1", title: "Demo test case", test: async()=> {
+			await context.ethers().sendTransaction({to:BALTATHAR_ADDRESS, value: 10n * GLMR });
+			await context.createBlock();
```

Comment:
would be good to add an  `expect()` here too to make it a complete test. It would be good to introduce users to explicitly importing expect from moonwall, which is not the case with mocha (where it's added automatically to the global scope as standard)